default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, linuxPackages
+, kernel ? linuxPackages.kernel
+, ...
+}:
+stdenv.mkDerivation rec {
+	pname = "msi-ec-kmods";
+	version = "2024-09-18";
+
+	src = fetchFromGitHub {
+		owner = "BeardOverflow";
+		repo = "msi-ec";
+		rev = "e5820a2b415e796db9dfb204250f7410b6662ac2";
+		hash = "sha256-1nwIf5OWjJpLLRUKeSOcZ1yvBGE51rUAZLmZjkt8K04=";
+	};
```

Comment:
```suggestion
	src = self;
```

Reply:
I added a development package with local sources in addition of the pinned package.
The pinned package should be kept anyway.

Reply:
> I added a development package with local sources in addition of the pinned package. The pinned package should be kept anyway.

I believe that it would make more sense to add the pinned version only to nixpkgs. What's the purpose of having the same thing in nixpkgs and here?

Reply:
Till it is not added to nixpkgs, it can be reffered from this flake with pinned version